NOTICE-binary

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------------------------------------
+// NOTICE file corresponding to the section 4d of The Apache License,
+// Version 2.0, in this case for Apache Kyuubi
+// ------------------------------------------------------------------
+
+Apache Kyuubi
+Copyright 2021 and onwards The Apache Software Foundation.
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+=====================================================================
```

Comment:
These text will be the first part in final `NOTICE-binary`

Reply:
Run `build/release/create-package.sh binary` and unarchive tgz to check the final `NOTICE-binary`